### ISSUE TITLE
It seems like a recent commit caused some breakage

### ISSUE THREAD

**Issue by jinzhongjia** [jdtsmith/indent-bars#66]
But when my emacs starts, it reports this

```
Args out of range: #<buffer *scratch*>, 8, 9
```

I don't know what else to provide, if you need more info please tell me

**Comment by jdtsmith** [comment]
Can you `M-x toggle-debug-on-error` and show the traceback?

**Comment by shipmints** [comment]
Can confirm.

```elisp
Debugger entered--Lisp error: (args-out-of-range #<buffer *scratch*> 8 9)
  match-string(1)
  #f(compiled-function (x) #<bytecode -0x70909c1b0f8c70f>)(outline-8)
  mapcan(#f(compiled-function (x) #<bytecode -0x70909c1b0f8c70f>) (rainbow-delimiters-depth-1-face rainbow-delimiters-depth-2-face rainbow-delimiters-depth-3-face rainbow-delimiters-depth-4-face rainbow-delimiters-depth-5-face rainbow-delimiters-depth-6-face rainbow-delimiters-depth-7-face rainbow-delimiters-depth-8-face rainbow-delimiters-depth-9-face rainbow-delimiters-mismatched-face rainbow-delimiters-unmatched-face rainbow-delimiters-base-error-face rainbow-delimiters-base-face posframe-border marginalia-file-priv-rare marginalia-file-priv-other marginalia-file-priv-exec marginalia-file-priv-write marginalia-file-priv-read marginalia-file-priv-link marginalia-file-priv-dir marginalia-file-priv-no marginalia-file-owner marginalia-file-name marginalia-modified marginalia-string marginalia-number marginalia-size marginalia-installed marginalia-archive marginalia-version marginalia-date marginalia-mode marginalia-list marginalia-symbol marginalia-function marginalia-true marginalia-null marginalia-value marginalia-documentation marginalia-off marginalia-on marginalia-lighter marginalia-char marginalia-type marginalia-key vertico-current vertico-group-separator vertico-group-title vertico-multiline ...))
  indent-bars--colors-from-regexp("outline-\\([0-9]+\\)" nil)
  indent-bars--depth-palette(#s(indent-bars-style :tag nil :main-color "#27ce1fa50000" :depth-palette nil :faces nil :stipple-face indent-bars-face :no-stipple-chars nil :current-stipple-face nil :current-bg-color nil :current-depth-palette nil))
  indent-bars--initialize-style(#s(indent-bars-style :tag nil :main-color "#27ce1fa50000" :depth-palette nil :faces nil :stipple-face indent-bars-face :no-stipple-chars nil :current-stipple-face nil :current-bg-color nil :current-depth-palette nil))
  indent-bars-setup()
  indent-bars-mode()
  (progn (indent-bars-mode))
```

**Comment by jinzhongjia** [comment]
Oh, thank you for your report. I haven’t done the debugging yet because I have other things to do.

**Comment by jdtsmith** [comment]
Thanks, stupid bug I introduced.  Should be fixed in master and v0.7.6 (please confirm).

**Comment by shipmints** [comment]
Pulled master. Seems good to me.
